test/UniswapV3PriceFeed.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { FakeContract, smock } from "@defi-wonderland/smock"
+import { expect } from "chai"
+import { BigNumber } from "ethers"
+import { parseEther } from "ethers/lib/utils"
+import { ethers, waffle } from "hardhat"
+import { UniswapV3Pool, UniswapV3PriceFeed } from "../typechain"
+
+interface UniswapV3PriceFeedFixture {
+    uniswapV3PriceFeed: UniswapV3PriceFeed
+    uniswapV3Pool: FakeContract<UniswapV3Pool>
+}
+
+async function uniswapV3PriceFeedFixture(): Promise<UniswapV3PriceFeedFixture> {
+    const [admin] = await ethers.getSigners()
+    const uniswapV3Pool = await smock.fake<UniswapV3Pool>("UniswapV3Pool", admin)
+
+    const uniswapV3PriceFeedFactory = await ethers.getContractFactory("UniswapV3PriceFeed")
+    const uniswapV3PriceFeed = (await uniswapV3PriceFeedFactory.deploy(uniswapV3Pool.address)) as UniswapV3PriceFeed
+
+    return { uniswapV3PriceFeed, uniswapV3Pool }
+}
+
+describe("UniswapV3PriceFeed Spec", () => {
+    const [admin] = waffle.provider.getWallets()
+    const loadFixture: ReturnType<typeof waffle.createFixtureLoader> = waffle.createFixtureLoader([admin])
+    let uniswapV3PriceFeed: UniswapV3PriceFeed
+    let uniswapV3Pool: FakeContract<UniswapV3Pool>
+
+    it("force error, pool address has to be a contract", async () => {
+        const uniswapV3PriceFeedFactory = await ethers.getContractFactory("UniswapV3PriceFeed")
+        await expect(uniswapV3PriceFeedFactory.deploy(admin.address)).to.be.revertedWith("UPF_PANC")
+    })
+
+    describe("", () => {
```

Comment:
```suggestion
    describe("", () => {
```
remove the redundant describe wrapper

Reply:
"pool address is contract"